build-parent/pom.xml

```diff
@@ -481,14 +481,12 @@
                                             <exclude>io.rest-assured:*</exclude>
                                             <exclude>org.assertj:*</exclude>
                                             <exclude>org.junit.jupiter:*</exclude>
-                                            <exclude>org.testcontainers:*</exclude>
```

Comment:
I suppose you removed it because it pops up as a transitive dependency in many places now?
I wonder whether we can keep it by making the rule non-transitive? This would still cover the main problem that #13919 tried to address.

Reply:
I would rather keep it consistent. If we only installed it for JDK8 it would just be one more difference between the environments. getPackage was deprecated in 9 anyway.